### DISCUSSION TITLE
Infinite importing notion zip

### DISCUSSION THREAD

**Discussion by RomanMazyrin** [outline/outline#4044] [Self Hosting]
I ve started import of 6kb zip archive 2 hours ago, and it is still in process. I suppose there is some trouble with import, but i can not even remove this task from UI. By the way, that archive successfully uploaded to my S3 storage (minio), i ve checked, it is all right.

What am i supposed to do? Where can i look at tasks queue?

![image](https://user-images.githubusercontent.com/52980260/187723960-3938565b-f876-46c6-b19e-55f4a5478f4a.png)

**Comment by tommoor** [comment]
It should complete pretty much instantly, there is likely some sort of error that isn't caught correctly - would check the error logs

**Reply by RomanMazyrin** [reply]
There is no any errors. Here is output after i sent same file to import once again:

``` bash
outline_1       | 2022-08-31T20:53:35.940075646Z {"name":"fileOperations.create","modelId":"3da7f64e-8bb5-469d-b99f-5cdc321bd1f0","attempt":0,"label":"worker","level":"info","message":"Processing fileOperations.create"}
outline_1       | 2022-08-31T20:53:35.966762999Z {"name":"fileOperations.create","modelId":"3da7f64e-8bb5-469d-b99f-5cdc321bd1f0","label":"worker","level":"info","message":"FileOperationsProcessor running fileOperations.create"}
outline_1       | 2022-08-31T20:53:35.990528854Z {"fileOperationId":"3da7f64e-8bb5-469d-b99f-5cdc321bd1f0","label":"worker","level":"info","message":"ImportNotionTask running"}
outline_1       | 2022-08-31T20:53:35.993400974Z {"name":"fileOperations.create","modelId":"3da7f64e-8bb5-469d-b99f-5cdc321bd1f0","label":"worker","level":"info","message":"WebhookProcessor running fileOperations.create"}
outline_1       | 2022-08-31T20:53:36.048717442Z {"label":"task","level":"info","message":"ImportTask fetching data for 3da7f64e-8bb5-469d-b99f-5cdc321bd1f0"}
```

And that's all. Nothing after that happened and file is still in progress according to "Recent imports" page.

**Reply by RomanMazyrin** [reply]
My apologies, second import failed with an error some time afterwards.

```bash
{"error":"connect ETIMEDOUT ***(IP ADDRESS)","stack":"Error: connect ETIMEDOUT ***(IP ADDRESS)\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16)","level":"error","message":"Error getting file from S3 by key"}
```

**Reply by RomanMazyrin** [reply]
But anyway, first import still pending, i just want to know any way to remove it from the import list

**Reply by tommoor** [reply]
Yea, It's not supposed to be possible so there isn't UI for this – you'd just need to edit the state of the fileOperation directly in the database.

**Reply by RomanMazyrin** [reply]
Okay, got it, i m gonna try to do something
